base-system/tools/installer/install.sh

```diff
@@ -95,6 +113,22 @@ while [ "$#" -gt 0 ]; do
 	esac
 done
 
+if [ -z "$INSTANCE_IP_ADDRESS" ]; then
```

Comment:
You can simplify the condition to:
if (any of this vars is non-empty) and (any of this vars is empty); then ...

Reply:
Thanks for the suggestion, added.